test/JsonApiDotNetCoreTests/IntegrationTests/QueryStrings/QueryStringTests.cs

```diff
@@ -63,6 +63,24 @@ public async Task Can_use_unknown_query_string_parameter()
         httpResponse.ShouldHaveStatusCode(HttpStatusCode.OK);
     }
 
+    [Theory]
+    [InlineData("")]
+    [InlineData("foo")]
```

Comment:
```suggestion
    [InlineData("bar")]
```
To match up with the test above it.